generators/client/templates/angular/_package.json

```diff
@@ -123,7 +123,7 @@
     "e2e": "protractor <%= TEST_SRC_DIR %>protractor.conf.js",
     "postinstall": "webdriver-manager update && <%= clientPackageManager %> run webpack:build"
     <%_ } else { _%>
-    "postinstall": "<%= clientPackageManager %> run webpack:build"
+    "postinstall": "node node_modules/pngquant-bin/lib/install.js && node node_modules/phantomjs-prebuilt/install.js && <%= clientPackageManager %> run webpack:build"
```

Comment:
@ruddell : should we add this `node node_modules/pngquant-bin/lib/install.js && node node_modules/phantomjs-prebuilt/install.js` to postInstall for protractor too ?